go/chat/unfurl/scraper.go

```diff
@@ -15,13 +16,15 @@ type Scraper struct {
 	utils.DebugLabeler
 	cache      *unfurlCache
 	giphyProxy bool
+	libkb.Contextified
 }
 
-func NewScraper(logger logger.Logger) *Scraper {
+func NewScraper(g *libkb.GlobalContext, logger logger.Logger) *Scraper {
```

Comment:
Same comment here, you can use the logger on `G` instead of the one that gets passed in,

Reply:
Done.